python_anticaptcha/base.py

```diff
@@ -147,9 +160,19 @@ def getBalance(self):
         return response['balance']
 
     def reportIncorrectImage(self, task_id):
-        request = {"clientKey": self.client_key,
-                   "taskId": task_id
-                   }
+        request = {
+            "clientKey": self.client_key,
+            "taskId": task_id
+        }
         response = self.session.post(urljoin(self.base_url, self.REPORT_IMAGE_URL), json=request).json()
         self._check_response(response)
         return response.get('status', False) != False
+                                                    
+    def reportIncorrectRecaptcha(self, task_id):
+        request = {
+            "clientKey": self.client_key,
+            "taskId": task_id
+        }
+        response = self.session.post(urljoin(self.base_url, self.REPORT_RECAPTCHA_URL), json=request).json()
+        self._check_response(response)
+        return response['status'] == "success"
```

Comment:
If Anti-captcha omit the required fields in response, I prefer to receive the error instead of arbitrarily taking the default values.